_includes/scripts.html

```diff
@@ -39,4 +36,64 @@
 </script>
 <!-- Intercom -->
 <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/rzcjacgd';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
+<script>
+  (function ($) {
+    var cookieInnerHtml = '<div><p>By using this website you agree to our <a href="/cookie-policy/">cookie policy</a></p><button id="cookieModalClose" class="btn btn-primary">OK</button></div>';
+
+    var getCookiebyName = function (name){
+      var pair = document.cookie.match(new RegExp(name + '=([^;]+)'));
```

Comment:
Can you add a comment explaining what you're doing here and an example of what you're trying to parse? It's possible to reverse engineer it from the regex, but why make maintainers do extra work?

Reply:
This cookie script was taken from the existing one that was found in main.js

Reply:
Yes, I understand that, but boyscout rule says: leave it cleaner than you found it.

Reply:
@brikis98 The getCookiebyName function uses Reg Ex to grab all the cookies, and is later used to check if _GruntyCookie_ already exists or not. It was inspired by this [stack overflow answer](https://stackoverflow.com/questions/5639346/what-is-the-shortest-function-for-reading-a-cookie-by-name-in-javascript), but in short:

> regex approach is not only the fastest in most browsers, it yields the shortest function as well. 

I think I was initially using a different way to check for cookies, but you had suggested I go the RegEx route, so we had gone around this way. 

Here, @drmzio simply took the code it out from main.js and placed it inside scripts to load it at a later time for it to be initiated after intercom scripts, so the cookie notice can appear on top with a higher Z-index.

Reply:
The StackOverflow link and the context you provided is exactly the kind of thing that makes for a great comment in the code.

Reply:
@sahilakos Can you add the a comment here as Jim is requesting?

Reply:
Oh, actually looks like you added a comment. Nothing further needed here!